EventEnabledInsurance/readme.md

```diff
@@ -559,3 +559,64 @@ DB_writer bar file: Responsible for Reading messages from the Queue `Quote` and
 
 # Testing the POST calls via APIC
 Instructions to load test the POST call via APIC can be found [here](post-load-test-readme.md).
+
+# Component Downtime Testing
+
+Prereqs:
+1. Configure kafka connectors
+2. Call REST endpoint post & get
+
+## I. Shutting down the db writer integration server
```

Comment:
Just for information, 0 replicas is supposed to be supported (but maybe not released yet) so at some point we'll be able to simplify these instructions. See:
https://ibm-cloud.slack.com/archives/CTANE7CAY/p1604665674192200?thread_ts=1604664709.191900&cid=CTANE7CAY